README.md

```diff
@@ -147,14 +147,14 @@ npm install -g @pnp/cli-microsoft365
 
 ## Usage
 
+>Before logging in, you should create a custom Entra Identity. Use the `m365 setup` command or refer to the [Using your own Microsoft Entra identity](https://pnp.github.io/cli-microsoft365/user-guide/using-own-identity/) guide.
```

Comment:
Throughout our guide we're referring to app registration while here we're using Entra Identity. Since we talk about creating a custom Entra Identity, I suggest we rephrase to "create a custom Entra app registration".

Reply:
As you mentioned below, I'm aiming for more consistency with the term naming. Wouldn't "app registration" be clear enough, or should we specify "Entra" in front of it? Based on the consistency in the MS docs, it’s commonly referred to as just "app registration," but those docs focus specifically on Entra Identity, so I'm not quite sure here. Ref: https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app?tabs=certificate

Reply:
Typically in Microsoft docs we handle it in such a way, that the first occurrence is full name, and then the following, when the context is clear, are shortened to avoid repetition. So in our example, the first time we could use "Entra application registration" (or even "Microsoft Entra" if really want to align with Microsoft branding guidelines) and then follow with "application registration".